src/main/options.js

```diff
@@ -117,13 +117,12 @@ function inferParser(filepath, plugins) {
     plugins
   }).languages.find(
     language =>
-      language.since !== null &&
-      (language.extensions.indexOf(extension) > -1 ||
-        (language.filenames &&
-          language.filenames.find(name => name.toLowerCase() === filename)))
```

Comment:
This `language.since` check is present to prevent the HTML and Handlebars parsers from being inferred, so we need to keep it in. Did you need to remove this to support a third-party plugin? I don't think third-party plugins need to ever define `since`, so you could take it out of your plugin (since this is looking for null specifically). Alternatively, we could add an `unreleased` property to the HTML and Handlebars languages, and check for that instead. I think that would be more clear.

Reply:
Many thanks for spotting this @suchipi! I guess `language.since !== null` disappeared just by my stupid mistake when I was guarding `language.extensions.indexOf`. There was a null pointer exception somewhere without this; all fixed now.

I did not run the tests locally because I'm looking into async `getFileInfo()` now – hope they pass in CI 😅